src/concat/source-map-combiner/combiner.js

```diff
@@ -0,0 +1,60 @@
+var Generator = require('./generator');
+var getMappings = require('./get-mappings');
+var convert = require('convert-source-map');
+
+function Combiner() {

```

Comment:
I've search around for the past few weeks for an existing npm module that does this, and couldn't find one that fits our needs. There were a bunch that were close, but didn't quite to the trick. This is just a simple wrapper around calls to the actual `source-map` library that allows us to concatenate source maps from multiple files into a single source map.